docs/quick_start.md

```diff
@@ -0,0 +1,195 @@
+    
+    *Note: for the latest release, please visit [here](https://github.com/napari/napari/releases) and look for Assets.*
+
+If you run into any issue, please visit the more detailed [installation guide](https://napari.org/tutorials/fundamentals/installation.html), or report an issue on GitHub!
+
++++
+
+### Open napari
+
+napari can be opened by [multiple ways](https://napari.org/tutorials/fundamentals/getting_started.html), depending on how its used in your image analysis workflow. 
+
+Here we will be mainly focused on the GUI application.
+
+- From command line: 
+
+    Once installed, simply run    
+```python
+napari
+```
+
+- If you installed the bundled app:
+
+    Click on the app icon to open it.<br>
+    *Note: for mac users, it may require [security setting changes](https://napari.org/tutorials/fundamentals/installation.html).*
+
++++
+
+### Open an image
+
+napari natively supports tiff and many other formats supported by [skimage.io.imread](https://scikit-image.org/docs/dev/api/skimage.io.html) as input image file format.<br>
+Try with your own images or download [this ome tiff file](https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/MitoCheck/00001_01.ome.tiff).
+
+Additional input file formats may be supported [by plugins](https://www.napari-hub.org/). 
+Try [napari-aicsimageio](https://www.napari-hub.org/plugins/napari-aicsimageio) if you have czi, lif, or nd2 files.
+
+Once you have the proper plugin installed, use File > Open Files(s)... and select the image file, or simply drag and drop the image into napari. 
+
+For demo purpose, we will use a sample image that comes with napari.
+
+(1) Open napari IPython console<br>
+<img src="img/IPython.png" alt="IPython console" width="30%"/>
+
+(2) Type
+    
+```python
+from skimage import data
+viewer.add_image(data.cell(), name='cell')
+```
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+
+import napari
+from napari.utils import nbscreenshot
+
+viewer = napari.Viewer()
+
+from skimage import data
+viewer.add_image(data.cell(), name='cell')
+nbscreenshot(viewer)
+```
+
+### Image display adjustment
+
+The layer control panel at the upper left of napari viewer allows you adjust contrast, colormap etc. These settings affect the visualization, but do not affect the underlying data.
+
+To change the image display through [API](https://napari.org/api/stable/napari.layers.Image.html), in IPython console, type
+
+```python
+viewer.layers['cell'].colormap = "yellow"
+```
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+
+viewer.layers['cell'].colormap = "yellow"
+nbscreenshot(viewer)
+```
+
+### Manually label the cell
+
+To measure the area of the cell, we can use labels layer and manually "paint" the cell.
+The labels layer allows you to record the segmentation result by assigning background = 0, and assigning each object with an integer.
+
+(1) Add a new labels layer <br> 
```

Comment:
I believe you can do a markdown numbered list with:

```md
1. Add a new labels layer
2. Click on "paint"
...
```

Reply:
In fact, if you use 
1.
1.
1.
in markdown, it will auto populate the numbers and makes later insertion really easy!